Server/Controllers/ShippingServiceController.cs

```diff
@@ -36,14 +41,50 @@ public async Task<ActionResult<Courier>> GetAsync(string courierId)
         return courier;
     }
 
-    [HttpPost("{CourierId}", Name = "PostAsync")]
+    [HttpPost("courier/{CourierId}", Name = "PostAsync")]
     [ActionName(nameof(PostAsync))]
     public async Task<ActionResult<Courier>> PostAsync([FromBody] Courier courier)
     {
         await _courierRepository.CreateAsync(courier);
 
         return CreatedAtRoute(nameof(PostAsync), new { id = courier._id }, courier);
     }
+    [HttpGet("new-packages", Name = "GetUnassignedPackages")]
+    [ProducesResponseType(StatusCodes.Status200OK)]
+    public async Task<ActionResult<List<Package>>> GetUnassignedPackages()
+    {
+        return await _packageService.GetUnassignedAsync();
+    }
+
+    [HttpPut("courier/{CourierId}/{PackageId}", Name = "AddCourierPackage")]
+    [ProducesResponseType(StatusCodes.Status200OK)]
+    [ProducesResponseType(StatusCodes.Status409Conflict)]
```

Comment:
TBH not quite familiar with 409 conflict, not often I see 409. Maybe it's because 409 shows some inner workings of the system? Cause then user could map your dependencies? Also, isn't it our job to make those request that could end up being 409 impossible, cause returning 409 means we have some inner problems?